### DISCUSSION TITLE
Any ideas how add nvim-ts-rainbow?

### DISCUSSION THREAD

**Discussion by KirovAI** [NvChad/NvChad#1458] [Q&A] [answered]
nvim-ts-rainbow is not just plugin for neovim, it plugin for treesitter which  we should attach when execute require "nvim-treesitter.configs" according to documentation.
Reference on this plugin: https://github.com/p00f/nvim-ts-rainbow
Beforehand thanks for all peoples which will help

**Comment by siduck** [comment]
@KirovAI you just need to load it after nvimtreesitter  and override treesitter config ( add the rainbow table in it )

**Reply by siduck** [reply, marked as answer]
@REDIDSOFT check dependencies feature of lazy.nvim
